FirebaseCombineSwift/Tests/Unit/Functions/HTTPSCallableTests.swift

```diff
@@ -0,0 +1,140 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import Combine
+import XCTest
+@testable import FirebaseFunctions
+
+class MockFunctions: Functions {
+  var mockCallFunction: () throws -> HTTPSCallableResult?
+  override func callFunction(_ name: String,
+                             with data: Any?,
+                             timeout: TimeInterval,
+                             completion: @escaping (HTTPSCallableResult?, Error?) -> Void) {
+    do {
```

Comment:
nit: I would recommend to validate the input parameters as well to make sure the Combine wrapper doesn't miss or mix anything.

Reply:
Added a callback that allows the test to verify the correct argument was sent.

Reply:
Thanks! Would you like to validate `name` and `timeout`?